src/Core/Request.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Tringuyen\Unlock\Core;
+
+class Request
+{
+    public function getPath()
+    {
+        $path = $_SERVER['REQUEST_URI'] ?? '/';
+        $position = strpos($path, '?');
+        if ($position === false) {
+            return $path;
+        } else {
```

Comment:
Should not using else.
About if else statement. We need to apply early return.
Example
```
if (!$position) {
   return $path;
}

return substr($path, 0, $position);
```